recording/docker-compose/dockerfile-talk-recording

```diff
@@ -0,0 +1,39 @@
+FROM phusion/baseimage:focal-1.2.0
+
+# https://github.com/nextcloud/spreed/blob/master/recording/start-container.sh
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+# Common dependencies
+RUN apt-get -y install jq wget
+
+# spreed-recording dependencies
+RUN apt-get -y install ffmpeg pulseaudio python3-pip xvfb
+
+# firefox
+RUN apt-get -y install firefox firefox-geckodriver
+
+# chromium
+RUN add-apt-repository -y ppa:phd/chromium-browser
+RUN printf "Package: *\nPin: release o=LP-PPA-phd-chromium-browser\nPin-Priority: 1001" > /etc/apt/preferences.d/phd-chromium-browser
+RUN apt-get -y install chromium-browser chromium-chromedriver
```

Comment:
I guess the PPA is needed due to Chromium not being updated to latest versions in Ubuntu Focal, is that right? In that case, could you add a comment about that in the Dockerfile?

Reply:
Lately all the browsers have been transitioning to snap packages in Ubuntu. It is not a trivial task to get those installed in Docker.

The PPA in question offers the latest non-snap Chromium available for Ubuntu 18.04, but built for newer Ubuntu versions.

Will add a comment to the Dockerfile.

Reply:
> Lately all the browsers have been transitioning to snap packages in Ubuntu. It is not a trivial task to get those installed in Docker.

Oh, I very well know... I wast... _spent_ a lot of time to build Ubuntu packages for the recording server _thanks_ to that transition to Snap packages :-P

> The PPA in question offers the latest non-snap Chromium available for Ubuntu 18.04, but built for newer Ubuntu versions.

Perfect, thanks for the clarification.

Reply:
Now that I see it the PPA is not an official PPA but something maintained by individuals, isn't it? I would prefer to use something _official_ (I mean, not necessarily something directly coming from Chromium team, the official package of a Debian based distribution not using Snaps would be fine as long as it is updated), but I do not know if there is something like that for Chromium :thinking: (but I guess no and that is why the phd PPA was used in first place ;-) ).

If I am not mistaken a package of Chrome (rather than Chromium) is provided by Google, but... in that case I would rather use a community version of the free (as in freedom) Chromium rather than an official version of the closed Chrome.

Reply:
There are official Debian DEBs, but they're often outdated if one is not using the _Sid_ rolling release. That PPA is the only one I've found so far that works for Ubuntu+ARM64, and is a simple&clean rebuild.

Fedora and Enterprise Linux 9 offer an up-to-date non-snap package too, but it would imply moving away from Ubuntu.

As you've mentioned, it is not an official PPA. It is mantained by:
- https://askubuntu.com/users/1531400/piotr-henryk-dabrowski
- https://launchpad.net/~phd